setup.cfg

```diff
@@ -26,81 +26,6 @@ ignore = W504
 follow_imports = silent
 ignore_missing_imports = True
 
-[mypy-pip/_internal/build_env]
-strict_optional = False
-
-[mypy-pip/_internal/cache]
-strict_optional = False
-
-[mypy-pip/_internal/cli/base_command]
-strict_optional = False
-
-[mypy-pip/_internal/cli/cmdoptions]
-strict_optional = False
-
-[mypy-pip/_internal/configuration]
-strict_optional = False
-
-[mypy-pip/_internal/index]
-strict_optional = False
-
-[mypy-pip/_internal/legacy_resolve]
-strict_optional = False
-
-[mypy-pip/_internal/locations]
-strict_optional = False
-
-[mypy-pip/_internal/models/format_control]
-strict_optional = False
-
-[mypy-pip/_internal/operations/check]
-strict_optional = False
-
-[mypy-pip/_internal/operations/freeze]
-strict_optional = False
-
-[mypy-pip/_internal/operations/prepare]
-strict_optional = False
-
-[mypy-pip/_internal/pep425tags]
-strict_optional = False
-
-[mypy-pip/_internal/req/*]
-disallow_untyped_defs = True
```

Comment:
Can we keep this until it is set globally and exceptions inlined?

Reply:
We can. Is it okay if I don't?

I'm planning on merging this now and filing the follow up, setting disallow-untyped-defs = True globally in a few minutes.

Reply:
OK by me.